meetup_description

```diff
@@ -1,6 +1,7 @@
-Have you ever made changes to a project and wanted to go back, but lost track of where it was? Are you looking to start coding and want to save projects like a professional? Do you want to collaborate on a project?
+Interested in tech, but think it's not for you? Do you rarely see people who look like you coding? Afraid you missed your chance by not starting to code at 8?
+Then Girl Develop It is for you. This MeetUp is one of our featured introductory classes that will show you that coding is for everyone. Whether you are ready to start your own projects, improve your career or just want to communicate better with the coders in your life, we will help you with the basics.
 
-This class is all about how to get started with "Git" & "GitHub". Git is a tool to track versions of a project. GitHub is a site to host the project. It is a great place to start all your projects to be able to share, collaborate and review the history.
+This 3 hour class is about how to get started with "Git" & "GitHub". Git is a tool to track versions of a project. GitHub is a site to host the project. It is a great place to start all your projects to be able to share, collaborate and review the history.
```

Comment:
Maybe reword this second sentence.
->
"It's a great resource that makes it easy to share, collaborate and review the history of your project.